Fedora/README.md

```diff
@@ -16,21 +19,30 @@
 ``(Stability , Security , Performance , Compatible apps , Hardware/System requirements, Technical support of the distro's maintainers) and any other points you find``
 
 ## Who is it made for?
-``-> Add distros targeted users``
+Fedora is targeted at the average user that isn't necessarily tech savy however it can still be used by more advanced linux users.
 ## Who is it **not** made for?
 ``-> Add types of users that **wont** find this distro suitable``
 
 ## What desktop environment does it use?
-``-> Add the desktop enviroment that it uses (gnome,kde,xfce,etc)``
+- Fedora by defualt uses GNOME DE but you can install other DEs manually later
 
 ## What package manager does it use?
 ``-> Add the package manager that it uses (apt, rpm, dpkg, etc)``
 ## User Experiences/Reviews
-``-> Add nice online user experiences/reviews that someone posted on this distro (not too long ones)``
+![Alt text](resources/user-review.png)
 
+
+You can view more reviews [here](https://www.capterra.com/p/229028/Fedora-Workstation/reviews/#:~:text=Pros%3A%20I've%20been%20a,best%20Linux%20distro%20available%20today.)
 ## Fun facts
 ``-> Add some fun facts about the distro like (Famous people who use the distro and Distrowatch ranking) ``
 ## How to install it?
-``-> Add instructions on methods of installing this distro``
+- You can get Fedora's ISO image [here](https://getfedora.org/en/workstation/download/)
+- You can then burn the ISO image to a USB stick using [Balena Etcher](https://www.balena.io/etcher) or [Rufus](https://rufus.ie/en/)
+- After creating bootable usb stick, Restart your device and boot into USB stick
+- You can try out fedora before installing it on you hard drive
```

Comment:
**Typo**, You can try out fedora before installing it on **you**->**your** hard drive

Reply:
typo aside, I also need to add images